client/src/Home.js

```diff
@@ -335,8 +336,37 @@ function Home() {
             )}
           </div>
         </div>
+        <div className="row my-5">
+          <div className="col-lg-6 text-center">
+            <Histogram
+              path="users"
+              field="age"
+              bin_custom="18,30,45,61"
```

Comment:
In the Asana task, Danielle describes the buckets as:

```
Desired age groups: 18-29, 30-44, 45-59, 60+
```
So should the last bin_custom value just be 60...?